ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -7,28 +7,38 @@
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
-    - name: Call predeploy
+    - name: Perform Podified and EDPM deployment on compute nodes provisioned with bmaas
+      when: edpm_compute_baremetal | default('false') | bool
```

Comment:
cosmetic and convention: would be good to prefix *all* of the parameters with `cifmw_`. That way, they will be gathered and written in a specific artifacts, as described here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/01-bootstrap.yml#L31-L39

Reply:
done.